datalad/support/external_versions.py

```diff
@@ -42,24 +42,31 @@ def __cmp__(self, other):
 #
 # Custom handlers
 #
-from datalad.cmd import Runner
-from datalad.cmd import GitRunner
+from datalad.cmd import (
+    WitlessRunner,
+    GitWitlessRunner,
+    StdOutErrCapture,
+)
 from datalad.support.exceptions import (
     MissingExternalDependency,
     OutdatedExternalDependency,
 )
-_runner = Runner()
-_git_runner = GitRunner()
+_runner = WitlessRunner()
+_git_runner = GitWitlessRunner()
 
 
 def _get_annex_version():
     """Return version of available git-annex"""
     try:
-        return _runner.run('git annex version --raw'.split())[0]
+        return _runner.run(
+            'git annex version --raw'.split(),
+            protocol=StdOutErrCapture)['stdout']
     except CommandError:
         # fall back on method that could work with older installations
-        out, err = _runner.run(['git', 'annex', 'version'])
-        return out.split('\n')[0].split(':')[1].strip()
+        out = _runner.run(
+            ['git', 'annex', 'version'],
+            protocol=StdOutErrCapture)
+        return out['stdout'].split('\n')[0].split(':')[1].strip()
```

Comment:
`splitlines` rather than `split('\n')`?

Reply:
I didnt want to make changes I wasnt sure about. It should do the same, but the original author didnt use it, and it wasnt a problem so far.

Reply:
Well, on windows the result of that split should end with `\r` now. Which in my estimation is a not yet unveiled issue rather than a non-existing one. ;-)
Anyways - not a blocker.

Reply:
Fair point!

Reply:
Done